src/clims/services/substance.py

```diff
@@ -365,11 +367,25 @@ def load_file(self, organization, full_path, fileobj, add_timestamp=False):
             full_path, ext = full_path.rsplit(".", 1)
             full_path = full_path + datetime.now().strftime("%m_%d_%Y%_H_%M_%S") + "." + ext
 
-        from sentry.plugins import plugins
-        plugins.require_handler(SubstancesSubmissionHandler)
+        context = HandlerContext(organization=organization)
+        with MultiFormatFile.from_file_stream(full_path, file_stream) as wrapped_stream:
+            plugins.handle(SubstancesValidationHandler, context, False, wrapped_stream)
```

Comment:
I think it could be useful for both the handle methods here to assign the parameters with keywords. Right now it is difficult to understand especially what `False` and `True` is actually setting.

Reply:
Oh, actually I had the same thought. I'll look into it. The reason for I didn't do anything was, on a deeper level there is an \*args in the signature after the True/False arg.

Reply:
That shouldn't be a problem I think, as long as all the other parameters are given as kwargs.